Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,26 +1,59 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Inventory
     {
-        private Item[] _items;
-        public Item[] GetItems()
+        public List<Item> Items { get; set; }
+
+        public Inventory()
         {
-            return new Item[0];
+            Items = new List<Item>();
         }
 
-        public Inventory()
+        /// <summary>
+        /// Returns the item it found based on the name you gave, if no item is found returns null
+        /// </summary>
+        /// <param name="name"></param>
+        /// <returns></returns>
+        public Item Contains(string name)
```

Comment:
Just from a naming perspective this should probably be GetItem, the singular version of your `GetItems()`.

Most implementations of `Contains` usually returns a bool to say "yes this collection contains this item/No this collection does not contain this item".

More of an FYI than a "you must fix" comment :)